index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
#### Good job working on setting up a 'Hello Microverse' project 👍. However, you need to fix some parts :

- [ ] Currently, the project doesn't have a linter installed as it is [required](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md#:~:text=Set%20up%20a%20linter%20for%20HTML%20and%20CSS%20and%20make%20sure%20that%20any%20files%20that%20are%20not%20relevant%20are%20ignored%20by%20git..), Kindly visit this [link](https://github.com/microverseinc/linters-config/tree/master/html-css) and follow the steps to install linters 👍  
- [ ] Also currently your project doesn't have a descriptive README as it is [required](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md#:~:text=Add%20descriptive%20README%20file%20to%20your%20project%20%2D%20please%20use%20this%20template%20but%20remember%20to%20customize%20it%20to%20your%20project.), Kindly us this [README temple](https://github.com/microverseinc/readme-template) and customize it to make it more descriptive 👍
- [ ]  Also, make sure your pull request is more descriptive, you may add more information about what you have done in this project 👍